app/src/main/java/com/nextcloud/talk/adapters/messages/IncomingPreviewMessageViewHolder.java

```diff
@@ -49,7 +54,36 @@ public IncomingPreviewMessageViewHolder(View itemView, Object payload) {
     @Override
     public void onBind(@NonNull ChatMessage message) {
         super.onBind(message);
-
+        if(!message.isVoiceMessage()) {
+            assert viewThemeUtils != null;
```

Comment:
this will crash when not null, so I would suggest to implement it more defensive, as in skip if null, applies also for the below uses of `assert` where I would advise against in general.